Add pigz support to vzctl create

mrqwer88 opened the pull request:
Add pigz support to vzctl create - https://bugs.openvz.org/browse/OVZ-5805

kolyshkin commented on scripts/vps-create.in:
```diff
@@ -56,6 +56,8 @@ create_prvt()
 		' 1f 8b') # gzip
 			OPT=-z
+			# Use pigz for more fast uncompress template if we can
+			pigz -V 2>&1 && TAR_OPT="--use-compress-program=pigz ${TAR_OPT}" && OPT=""
```
It should be `pigz -V >/dev/null 2>&1` I believe

mrqwer88 replied:
Yes - fixed it.

kolyshkin commented:
Also, can you please add some performance comparison to commit message?

mrqwer88 committed:
Add pigz support to vzctl create

It can speedup unpack template about 2 and more times.
As example runnig vzctl create with custom 702M size template:
(second and third run vzctl create with this template, file cache already warmed)
 - without pigz
    702MB 0:00:14 [49.4MB/s]

 - with pigz
    702MB 0:00:07 [90.9MB/s]

mrqwer88 commented:
Add pv output from vzctl create with time and speed to commit message.

kolyshkin commented:
Sergey, thank you! Merged.
